static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -60,6 +63,7 @@ type Props = {
   dataset: MetricRule['dataset'];
   environment: string | null;
   handleMEPAlertDataset: (data: EventsStats | MultiSeriesEventsStats | null) => void;
+  isQueryValid: boolean;
```

Comment:
Sorry, I can't visualize this in my head - will this still work (show chart) for orgs with `alert-allow-indexed` feature flag?